client/test/circuits/combo/max128.circuit.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  add,
+  addToCallback,
+  CircuitValue,
+  constant,
+  getAccount,
+  getHeader,
+  getReceipt,
+  getSolidityMapping,
+  getStorage,
+  getTx,
+  mul,
+} from "@axiom-crypto/circuit";
+
+export interface CircuitInputs {
+  blockNumber: CircuitValue;
+  contractBlock0: CircuitValue;
+  contractBlock1: CircuitValue;
+  contractBlock2: CircuitValue;
+  contract0: CircuitValue;
+  contract1: CircuitValue;
+  contract2: CircuitValue;
+}
+
+export const defaultInputs = {
+  blockNumber: 4000000, //$ account.eoa[7].blockNumber
+  contractBlock0: 4000000, //$ storage.nonzero[7].blockNumber
+  contractBlock1: 4000000, //$ storage.nonzero[8].blockNumber
+  contractBlock2: 4000000, //$ storage.nonzero[9].blockNumber
+  contract0: "0xEaa455e4291742eC362Bc21a8C46E5F2b5ed4701", //$ storage.nonzero[7].address
+  contract1: "0xEaa455e4291742eC362Bc21a8C46E5F2b5ed4701", //$ storage.nonzero[8].address
+  contract2: "0xEaa455e4291742eC362Bc21a8C46E5F2b5ed4701", //$ storage.nonzero[9].address
+};
+
+export const circuit = async (inputs: CircuitInputs) => {
+  for (let i = 0; i < 32; i++) {
+    let tx = getReceipt(inputs.blockNumber.number() - 50 + i, 0);
```

Comment:
for this to be an accurate circuit, these should use `add(inputs.blockNumber,i)` etc. As noted in the [docs](https://docs.axiom.xyz/sdk/typescript-sdk/axiom-circuit/circuit-types), `.number()` should only be used for debugging purposes.

Please check and address these systematically throughout.

Reply:
the `-50` and `10 +` below were some artifacts of the input selection: are they still needed?